src/opnsense/mvc/app/controllers/OPNsense/Kea/forms/dialogReservation4.xml

```diff
@@ -68,7 +68,7 @@
         <id>reservation.option_data.domain_name</id>
         <label>Domain name</label>
         <type>text</type>
-        <help>The domain name to offer to the client, set to this firewall's domain name when left empty</help>
+        <help>The domain name to offer to the client, set to interface's domain name or this firewall's domain name when both left empty</help>
```

Comment:
I think explaining the code here in the help text will both be confusing and prone to bitrot

```suggestion
        <help>A custom domain name to offer to the client.</help>
```